src/vsg/platform/win32/Win32_Window.cpp

```diff
@@ -149,32 +149,32 @@ KeyboardMap::KeyboardMap()
             {VK_RIGHT                             ,              KEY_Right    },
             {VK_DOWN                              ,              KEY_Down     },
             {VK_SELECT                            ,              KEY_Select},
-            {VK_PRINT                             ,              KEY_Print},
+            {VK_SNAPSHOT                          ,              KEY_Print},
```

Comment:
If VK_PRINT shouldn't map to KEY_Print would should VK_SNAPSHOT?

What does VK_PRINT and KEY_Print meaning and are you assuming?

Reply:
OK that's a good/valid question..
- I assumed KEY_Print would be the 'Print screen' hotkey/functionality.
- while the MSDN lists the VK_PRINT constant all it says is that it is the 'PRINT key' (I searched around a bit but I could only find [this stackoverflow discussion](https://stackoverflow.com/questions/5815471/whats-the-purpose-of-vk-print) where it's stated that VK_PRINT is basically a nowadays unused remnant from 83/84 key IBM XT/AT machines). However looking at they keyboard in front of me, pressing the 'PrtSc' button I could see that the VK_SNAPSHOT event was created by Windows :)

Reply:
I just tried under Linux and pressing "Press Screen" is mapping to VK_PRINT, so if Windows generates VK_SNAPSHOT for this key then that would be correct mapping.